src/ui/ReactionButton/index.tsx

```diff
@@ -28,6 +31,14 @@ const ReactionButton = React.forwardRef((props: ReactionButtonProps, ref: RefObj
     children,
   } = props;
 
+  const onClickHandler = useLongPress({
+    onLongPress: noop,
+    onClick: onClick,
```

Comment:
(Don't need to apply in this PR. I'm just commenting as a reminder for myself)

```suggestion
    onClick,
```

Could be same too. Let me turn on object-shorthand eslint rule later :) 
https://github.com/sendbird/sendbird-uikit-react/blob/main/.eslintrc.json#L81